packages/angular-query-experimental/src/__tests__/inject-queries.test.ts

```
@@ -1,28 +1,65 @@
+  ApplicationRef,
+  ChangeDetectionStrategy,
   Component,
+  Injector,
+  computed,
   effect,
-  provideZonelessChangeDetection,
+  input,
+  signal,
 } from '@angular/core'
 import { TestBed } from '@angular/core/testing'
+import { sleep } from '@tanstack/query-test-utils'
```

ESLint

[error] '@tanstack/query-test-utils' imported multiple times.

(import/no-duplicates)